solana-labs/solana-program-library
# Add anchor-build action

- rounding error fix (#51)
- adding confidence and status checks for pyth (#52)
- One oracle (#53)
- Update token lending readme (#54)
- Fix README
- update-readme-addreses (#56)
- allow oracle to be more stale (#57) (#58)
- dupe lite refresh (#62) (#63)
- fix ci solana version (#67)
- Update switchboard-program to v0.2.0 (#66)
- oracleless repay (#64)
- Upcoming (#68)
- remove unused programs (#60)
- Oracleless ctoken mint and redeem (#69)
- update liquidation close factor as part of param updates (#74)
- added liquidate and redeem (with protocol fee) (#75)
- repay floor to ceil hotfix
- adding switchboard v2 support (#73)
- fix init reserve comments (#81)
- Publish crate2 (#80)
- fix max withdraw (#82)
- allow loan_to_value_ratio == liquidation_threshold (#85)
- high rate map (#86)
- adding more high rate cases
- fix to support higher exponent for switchboard v2 (#90)
- trying take rate other way (#89)
- sliding down borrow limit and 1 percent close factor (#91)
- only redeem as much collateral as liquidity allows during liquidate (#88)
- undo borrow limit and close factor change and cap liquidation amount (#92)
- test coverage github action (#94)
- enable github actions on commits to upcoming branch (#96)
- turn on liquidation fees and deprecate old liquidation instruction (#100)
- bump solana to 1.9.18 (#97)
- flash loans (#95)
- bump cargo version (#103)
- make cli update only update if actually changing something and add an… (#101)
- allow 0 ltv assets to be deposited + minor cleanup (#104)
- Obligation_owner should be readonly (#87)
- allow passing just pyth to save bytes (#105)
- update debt limit (#107)
- 0xripleys manual liquidate (#112)
- fix cli for updating oracle (#114)
- Use pyth sdk + tests (#113)
- Create a crate, relax solana dependencies in crate (#118)
- bump solana version to 1.14 (#116)
- Refactor and improve tests (#121)
- changing util rate calc to exclude non-claimed fees (#132)
- Solend v2.0.1 (#131)
- div by zero fix (#138)
- fix for withdraw case (#140)
